webapp/channels/src/components/admin_console/schema_admin_settings.tsx

```diff
@@ -216,7 +216,21 @@ export class SchemaAdminSettings extends React.PureComponent<Props, State> {
                 schema.sections.map((section) => section.settings).forEach((sectionSettings) => settings.push(...sectionSettings));
             }
 
-            settings.forEach((setting) => {
+            // Recursively collect settings from expandable settings
+            const collectSettingsRecursively = (settingsArray: AdminDefinitionSetting[]): AdminDefinitionSetting[] => {
+                const allSettings: AdminDefinitionSetting[] = [];
+                settingsArray.forEach((setting) => {
```

Comment:
Nit: We might be able to do something a bit easier here by using `settings.flatMap` since that lets you lets you return any number of results for each item in the array and then it'll handle flattening that into a single array at the end, although that would probably require changes to the rest of this